client/checkout/blocks/confirm-upe-payment.js

```diff
@@ -23,6 +24,7 @@ export default async function confirmUPEPayment(
 	paymentIntentSecret,
 	elements,
 	billingData,
+	shippingAddress,
```

Comment:
The naming is mismatching, I'd expect something like `shippingData`.

Reply:
I'm not too deep into Blocks implementation. Hence please ensure that shipping details are presented in cases when a billing address is used for shipment and when a dedicated shipping address is specified.

Reply:
>The naming is mismatching, I'd expect something like shippingData.

I have fixed this here  - 514fa82

Reply:
>I'm not too deep into Blocks implementation. Hence please ensure that shipping details are presented in cases when a billing address is used for shipment and when a dedicated shipping address is specified.

I have manually tested this, and it seems to work fine. 

One point to be noted - In case of Block checkout, the default fields are `Shipping`, and we have an option of setting a different `Billing` address. This is opposite of how it appears on classic checkout.  In either case, the browser console as well as the test acknowledgement page of Afterpay show the expected data